hawkular-alerts-rest-tests/src/test/groovy/org/hawkular/alerts/rest/LifecycleITest.groovy

```diff
@@ -0,0 +1,250 @@
+/**
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.alerts.rest
+
+import org.hawkular.alerts.api.model.condition.AvailabilityCondition
+import org.hawkular.alerts.api.model.trigger.Trigger
+import org.hawkular.alerts.bus.messages.AlertData
+import org.hawkular.bus.restclient.RestClient
+
+import static org.hawkular.alerts.api.model.condition.AvailabilityCondition.Operator
+import static org.hawkular.alerts.api.model.trigger.Trigger.Mode
+import static org.junit.Assert.assertEquals
+
+import org.junit.Test
+
+/**
+ * Alerts REST tests.
+ *
+ * @author Jay Shaughnessy
+ */
+class LifecycleITest extends AbstractITestBase {
+
+    static host = System.getProperty('hawkular.host') ?: '127.0.0.1'
+    static port = Integer.valueOf(System.getProperty('hawkular.port') ?: "8080")
+
```

Comment:
In the PR description you said you wanted to be able to run the tests against your locally running server. You can do that by starting this test straight from your IDE. It will use the default host and port.
If that's not enough, then we can take what's been done in the Metrics project: there's a profile which, when activated, skips the Wildlfy start/deploy/stop goals.